Calibration/IsolatedParticles/plugins/IsoTrig.cc

```diff
@@ -550,7 +550,8 @@ void IsoTrig::fillDescriptions(edm::ConfigurationDescriptions &descriptions) {
   desc.addUntracked<double>("vertexCutIsol", 101.0);
   desc.addUntracked<double>("tauUnbiasCone", 1.2);
   desc.addUntracked<double>("prelimCone", 1.0);
-  descriptions.add("isoTrigHB", desc);
+  desc.add<unsigned int>("stageL1Trigger", 1);
```

Comment:
Why is the default the old "1" rather than the new/current "2"?

Reply:
The default for era modifiers (i.e. without any modifiers) corresponds run 1.